src/components/SurveyQuestion.js

```diff
@@ -0,0 +1,34 @@
+import classNames from "classnames";
+
+const SurveyQuestion = ({
+  question,
+  options,
+  selection,
+  onOptionClick,
+  onNextQuestion,
```

Comment:
I only meant to remove the `isSingleSelect` prop. The rest was fine. `isSingleSelect` made no sense since `isMultiSelect` was already there for the same reason.